ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-l2vlan-profile-does-not-exist.yml

```diff
@@ -0,0 +1,27 @@
+type: l3leaf
+
```

Comment:
I think we should also add l2vlan_profiles (with some other profile name) for more clarity.

Reply:
```suggestion
l2vlan_profiles:
  - profile: MYPROFILE
    name: vlan-name-from-profile
```